plugins/auth-backend/src/providers/oidc/provider.ts

```diff
@@ -56,24 +56,27 @@ type AuthResult = {
 export type Options = OAuthProviderOptions & {
   metadataUrl: string;
   scope?: string;
+  prompt?: string;
   tokenSignedResponseAlg?: string;
 };
 
 export class OidcAuthProvider implements OAuthHandlers {
   private readonly implementation: Promise<OidcImpl>;
   private readonly scope?: string;
+  private readonly prompt?: string;
 
   constructor(options: Options) {
     this.implementation = this.setupStrategy(options);
     this.scope = options.scope;
+    this.prompt = options.prompt;
   }
 
   async start(req: OAuthStartRequest): Promise<RedirectInfo> {
     const { strategy } = await this.implementation;
     return await executeRedirectStrategy(req, strategy, {
       accessType: 'offline',
-      prompt: 'none',
-      scope: req.scope || this.scope || '',
+      prompt: this.prompt || '',
```

Comment:
Just one change here, can we default it back to `none` so this becomes a non breaking change. We're doing different things than before (although before it might have been wrong) 👍

Reply:
That was my initial intent but then I wondered how this could have ever worked in the first place????
The prompt='none' is only used for a silent refresh, inside an invisible iFrame.
Do you really want me to keep 'none' or did you have a working use case?

Reply:
Yeah - I think so - I think there's reasons behind it being like that in the first place @Rugvip maybe knows more.

Reply:
ok, I will do the change to avoid any potential breaking change.
I guess that this should be better documented somewhere...

Reply:
Just pushed the fix. 
Note that I had to introduce the 'auto' value for prompt (instead of an empty string), since I can't rely on empty string to be the default behaviour any more. I also added more documentation in the app-config.yaml.
Anyway, I believe that it is better to have an explicit value instead of an empty string!